src/Redist/Microsoft.ML.TensorFlow.Redist/Microsoft.ML.Tensorflow.Redist.proj

```diff
@@ -0,0 +1,133 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Restore;Build">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
```

Comment:
Could this just be a `<Project>` (note without an Sdk), and just import `Directory.Build.props/targets` itself?  Then we wouldn't need to try to coerce the .NET.Sdk at all.

We could also just use the `Build` target, instead of overriding the `CoreCompile` target...  See `\src\Native\build.proj`.

Reply:
And we wouldn't need `CreateManifestResourceNames`.

---
In reply to: [212421508](https://github.com/dotnet/machinelearning/pull/720#discussion_r212421508) [](ancestors = 212421508)

Reply:
I'd lose the clean support then.  I guess I could write my own.

Initially I was trying to make this participate in the SLN so I wanted more of the normal targets, but I ran into issues with VS not liking the extension and `.msbuildproj` being broken with SDK-style projects.

I'll go ahead and get rid of the SDK.

---
In reply to: [212421920](https://github.com/dotnet/machinelearning/pull/720#discussion_r212421920) [](ancestors = 212421920,212421508)

Reply:
I don't know if this helps or hurts (so don't take this as a "I think you should do this"). I'm just throwing it out there for discussion purposes.

There is a ["NoTargets" Sdk](https://github.com/Microsoft/MSBuildSdks/tree/master/src/NoTargets) that is supposed to help in these types of scenarios.  I don't know how I feel about being dependent on this (I've never taken a dependency on it in a production project), but it is *supposed* to solve this scenario.

Reply:
I don't think we actually need it.  I took a look and it doesn't help much.  It doesn't pull in common targets so I'd still need to roll my own clean.  It also looks like the file extension it's using in samples is `csproj`.  Not sure I like that.  Feels cleaner to just use vanilla msbuild and let it be something that requires a full `build` of the repo before working.

---
In reply to: [212431038](https://github.com/dotnet/machinelearning/pull/720#discussion_r212431038) [](ancestors = 212431038)

Reply:
Sounds good to me.  Thanks for taking a look.